apps/mull-ui/src/app/pages/event-page/event-page.tsx

```diff
@@ -73,12 +74,11 @@ export const EventPage = ({
   }
 
   if (!loadingUser && dataUser) {
-    event.host = dataUser.user;
+    event.host = (dataUser.user as unknown) as IUser;
```

Comment:
It would be better if you set the type to `User` from the generated graphql

```suggestion
    event.host = (dataUser.user as unknown) as User;
```

And set the event to be of type `Event`, again from the generated graphql.

I also fix the typescript error in my pr. have a look at how I do it there

Reply:
Ah, I guess ill leave it as it was before and let your PR handle the types. I just got annoyed by the red lines here.